libopenage/engine.cpp

```diff
@@ -168,6 +168,19 @@ Engine::Engine(util::Dir *data_dir, const char *windowtitle)
 
 	glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
 
+	// qml sources will be installed to the asset dir
+	// otherwise assume that launched from the source dir
+	using namespace std::string_literals;
+	auto qml_search_paths = {this->data_dir->basedir, "libopenage/gui"s};
```

Comment:
Ok, I've looked into it and with `vector<string> qml_search_paths = {...}`, the implicit char* to string conversion works. But with `auto qml_search_paths = {...}`, the behaviour is more like with `template<typename T>`.